### DISCUSSION TITLE
Database Not Loading

### DISCUSSION THREAD

**Discussion by barnesb8** [uwrit/leaf#430] [Q&A] [answered]
I'm trying to run a development version on my local machine for testing, but I'm unable to connect the dev version to my own clinical database successfully. The only item I am seeing in the database dropdown is the example and I'm struggling to track down what's going wrong. Through some debugging, I can see that the API is using the appropriate connection string for the ClinDb, so I think my environment variables are correctly configured. Any ideas on where to look would be appreciated!

**Comment by ndobb** [comment, marked as answer]
Hi @barnesb8,

Based on your description, it sounds like things are actually working as expected.

The "Databases" dropdown (with default "University of Example") in the upper-right is actually designed for **users** to make sense of what system(s) they are querying, rather than administrators. In other words, it's just an admin-defined, user-facing window dressing to help orient them, and is decoupled from the actual clinical database you're connected to.

You can edit the text and colors by going to `Admin` -> `Network and Identity`:
![image](https://user-images.githubusercontent.com/10424962/124155318-37e3a400-da4b-11eb-9375-fc2fd993782d.png)

Then change the attributes as you like:
![image](https://user-images.githubusercontent.com/10424962/124155486-63ff2500-da4b-11eb-8c51-f98837322695.png)

To really test that your connection to the clinical database is working, however, you'll want to start by creating `Concepts` and trying to run queries with them. Please check out our docs tutorial on this at https://leafdocs.rit.uw.edu/administration/concepts/ and let us know if you have any other questions or issues.

Best,
-nic

**Reply by barnesb8** [reply]
Thanks! I didn't realize the text was a configuration so that was very helpful and my queries are working as expected like you suggested.
